ssh/DOCS.md

```diff
@@ -140,7 +140,8 @@ only apply to the SSH daemon.
 This option allows you to change to username the use when you log in via SSH.
 It is only utilized for the authentication; you will be the `root` user after
 you have authenticated. Using `root` as the username is possible, but not
-recommended.
+recommended. Usernames will be converted to lower case as per recommended
```

Comment:
It would be awesome if on saving the username in the config shown changes to lower case. Currently that is not the case. 

Currently you probably want to state it as: Usename specified must be lower case.

Reply:
Addons have not control over the UI on that level.